website/docs/quickstart/security.md

```diff
@@ -0,0 +1,510 @@
+        authorizer.enabled: true
+        super.users: User:admin
+    ports:
+      - "9123:9123"
+  tablet-server:
+    image: fluss/fluss:$FLUSS_VERSION$
+    command: tabletServer
+    depends_on:
+      - coordinator-server
+    environment:
+      - |
+        FLUSS_PROPERTIES=
+        zookeeper.address: zookeeper:2181
+        bind.listeners: INTERNAL://tablet-server:0, CLIENT://tablet-server:9123
+        advertised.listeners: CLIENT://localhost:9124
+        internal.listener.name: INTERNAL
+        tablet-server.id: 0
+        kv.snapshot.interval: 0s
+        data.dir: /tmp/fluss/data
+        remote.data.dir: /tmp/fluss/remote-data
+        # security properties
+        security.protocol.map: CLIENT:SASL, INTERNAL:PLAINTEXT
+        security.sasl.enabled.mechanisms: PLAIN
+        security.sasl.plain.jaas.config: com.alibaba.fluss.security.auth.sasl.plain.PlainLoginModule required user_admin="admin-pass" user_marketing="marketing-pass" user_finance="finance-pass";
+        authorizer.enabled: true
+        super.users: User:admin
+    ports:
+        - "9124:9123"
+    volumes:
+      - shared-tmpfs:/tmp/fluss
+  zookeeper:
+    restart: always
+    image: zookeeper:3.9.2
+
+volumes:
+  shared-tmpfs:
+    driver: local
+    driver_opts:
+      type: "tmpfs"
+      device: "tmpfs"
+```
+
+### Create Catalogs for Each User
+Create separate catalogs for the `admin`, `marketing`, and `finance` users:
+```sql title="Flink SQL"
+CREATE CATALOG admin_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'admin',
+'client.security.sasl.password' = 'admin-pass'
+);
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG marketing_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'marketing',
+'client.security.sasl.password' = 'marketing-pass'
+);
+
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG finance_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'finance',
+'client.security.sasl.password' = 'finance-pass'
+);
+```
+
+### Create Databases and Set ACLs
+As the `admin` user, create two databases and assign appropriate ACLs:
+```sql title="Flink SQL"
+CREATE DATABASE `admin_catalog`.`marketing`;
+CALL admin_catalog.sys.add_acl(
+    'cluster.marketing', 
+    'ALLOW',
+    'User:marketing', 
+    'ALL',
+    '*'
+);
+
+
+CREATE DATABASE `admin_catalog`.`finance`;
+CALL admin_catalog.sys.add_acl(
+    'cluster.finance', 
+    'ALLOW',
+    'User:finance', 
+    'ALL',
+    '*'
+);
+
+```
+
+### Granularity of Database Visibility
+
+The `marketing` user can only see the `marketing` database
```

Comment:
Name the database `marketing_db` to distinguish with the user name. The same to `finance`